packages/api/lib/analytics.js

```diff
@@ -68,30 +69,81 @@ class TestCaseAnalytics {
     return (this.mergedList.length === 0) ? 0 : numSuc / this.mergedList.length;
   }
 
-  // returns true if both...
-  // condition 1: there has been a failure within the last 5 runs
-  // condition 2: there have been any two failures separated by between 1 and 10 successes
-  computeIsFlaky () {
-    const failureIndices = [];
-    for (let k = 0; k < this.mergedList.length; k++) {
-      if (!this.mergedList[k].passed) {
-        failureIndices.push(k);
+  numberOfHistoricalTests () {
+    return (this.mergedList.length);
+  }
+
+  // returns true if there have been any two failures separated by between 1 and 14 successes
+  calculateFlaky () {
```

Comment:
a new function to compute a test's flakiness was needed, considering the fact that we are no longer keeping track of the past 15 test runs for every test in a repository